Kudu.Services/Deployment/PushDeploymentController.cs

```diff
@@ -354,13 +527,54 @@ bool isFunctionJson(string fullName)
             }
         }
 
-        private async Task<string> DeployZipLocally(ZipDeploymentInfo zipDeploymentInfo, ITracer tracer)
+        //Used by OneDeploy to fetch artifacts that are not ZIP files. OneDeploy ZIP files are handled with 
+        //LocalZipHandler()
+        private async Task OneDeployFetch(IRepository repository, DeploymentInfoBase deploymentInfo, string targetBranch, ILogger logger, ITracer tracer)
+        {
+            var artifactDeploymentInfo = (ArtifactDeploymentInfo)deploymentInfo;
+            //If ArtifactURL property is defined, the artifact is to be fetched from ArtifactURL. Otherwise, 
```

Comment:
// If RemoteUrl is non-null, it means the content needs to be downloaded from the Url source
// Else, it needs to be read from the request body